book/website/reproducible-research/reviewing/reviewing-recommend.md

```diff
@@ -1,56 +1,69 @@
-## Recommendations and best practices
+# Recommendations and best practices
 
-<a name="Be_nice"></a>
-### Be nice!
+## Who reviews?
+
+Within small-scale projects where the developers all typically already know each other,
+typical practice is for the coder to tag someone in the group as the reviewer.
+When you are tagged as a reviewer, first check if you have enough knowledge about
+the new changes in the code to do a good job in a reasonable amount of time.
+
+In contrast, large-scale development projects will likely have existing, concrete rules for
+how reviewers are allocated to individual pull requests.
+These rules serve to balance the group workload and to maximise the various benefits of the process to the project and its participants.
+The very largest projects may even have dedicated staff - or teams of staff - to act as reviewers.
+Typically, code reviews can only be performed by an authorised subset of contributors within larger projects.
+
+For projects where multiple rounds of review on similar material are likely and long development cycles are anticipated, a degree of strategic thinking on who completes reviews is sensible.
+A single reviewer is likely to be able to make comments on code they have reviewed before much more efficiently.
+However, letting reviewer-coder pairs like this ossify is generally a bad idea, as it can lead to the same kinds of groupthink that the review process is designed to avoid in the first place.
```

Comment:
```suggestion
However, letting reviewer-coder pairs like this persist is generally a bad idea, as it can lead to the same kinds of groupthink that the review process is designed to avoid in the first place.
```

I had to google "ossify" 😬 Hopefully this retains meaning while being a little more accessible?